changelog/upgrade-notes.md

```diff
@@ -6,6 +6,8 @@
 
 The deprecated `ol.animation` functions and `map.beforeRender()` method have been removed.  Use `view.animate()` instead.
 
+The deprecated `ol.Observable#unByKey` function has been removed.  Use `ol.Observable.unByKey` instead.
```

Comment:
This might seem a bit cryptic to some? Can we add some wording that the prototype / instance function has been removed here and that the static function should be used instead? It might not be obvious for everyone what the difference between ```ol.Observable#unByKey``` and ```ol.Observable.unByKey``` is?